pyfolio/risk.py

```diff
@@ -0,0 +1,486 @@
+    '''
+    Plots outputs of compute_sector_exposures as area charts
+
+    Parameters
+    ----------
+    long_exposures, short_exposures : arrays
+        Arrays of long and short sector exposures (output of
+        compute_sector_exposures).
+
+    sector_dict : dict or OrderedDict
+        Dictionary of all sectors
+        - See full description in compute_sector_exposures
+    '''
+
+    if ax is None:
+        ax = plt.gca()
+
+    if sector_dict is None:
+        sector_names = SECTORS.values()
+    else:
+        sector_names = sector_dict.values()
+
+    colors = ['#FF9999', '#FFCC99', '#FFFF99', '#CCFF99', '#99FF99', '#99FFCC',
+              '#99FFFF', '#99CCFF', '#9999FF', '#CC99FF', '#FF99FF']
+
+    ax.stackplot(long_exposures[0].index, long_exposures,
+                 labels=sector_names, colors=colors, baseline='zero')
+    ax.stackplot(long_exposures[0].index, short_exposures,
+                 colors=colors, baseline='zero')
+    ax.axhline(0, color='k', linestyle='-')
+    ax.set_title('Long and Short Exposures to Sectors')
+    ax.set_ylabel('Proportion of Long/Short Exposure in Sectors')
+    ax.legend(loc='upper left', frameon=True, framealpha=0.5)
+
+    return ax
+
+
+def plot_sector_exposures_gross(gross_exposures, sector_dict=None, ax=None):
+    '''
+    Plots output of compute_sector_exposures as area charts
+
+    Parameters
+    ----------
+    gross_exposures : arrays
+        Arrays of gross sector exposures (output of compute_sector_exposures).
+
+    sector_dict : dict or OrderedDict
+        Dictionary of all sectors
+        - See full description in compute_sector_exposures
+    '''
+
+    if ax is None:
+        ax = plt.gca()
+
+    if sector_dict is None:
+        sector_names = SECTORS.values()
+    else:
+        sector_names = sector_dict.values()
+
+    colors = ['#FF9999', '#FFCC99', '#FFFF99', '#CCFF99', '#99FF99', '#99FFCC',
+              '#99FFFF', '#99CCFF', '#9999FF', '#CC99FF', '#FF99FF']
+
+    ax.stackplot(gross_exposures[0].index, gross_exposures,
+                 labels=sector_names, colors=colors, baseline='zero')
+    ax.axhline(0, color='k', linestyle='-')
+    ax.set_title('Gross Exposure to Sectors')
+    ax.set_ylabel('Proportion of Gross Exposure \n in Sectors')
+
+    return ax
+
+
+def plot_sector_exposures_net(net_exposures, sector_dict=None, ax=None):
+    '''
+    Plots output of compute_sector_exposures as line graphs
+
+    Parameters
+    ----------
+    net_exposures : arrays
+        Arrays of net sector exposures (output of compute_sector_exposures).
+
+    sector_dict : dict or OrderedDict
+        Dictionary of all sectors
+        - See full description in compute_sector_exposures
+    '''
+
+    if ax is None:
+        ax = plt.gca()
+
+    if sector_dict is None:
+        sector_names = SECTORS.values()
+    else:
+        sector_names = sector_dict.values()
+
+    colors = ['#FF9999', '#FFCC99', '#FFFF99', '#CCFF99', '#99FF99', '#99FFCC',
```

Comment:
Actually, I prefer to have the colors occur in a rainbow: the reason being that if you were looking at a "patch" on the graph and wanted to look up sector in the legend, the order of the rainbow would help you quickly find the correct sector, whereas `Set3` orders colors in an essentially random order, forcing the reader to look through every legend entry and giving a greater cognitive load on the reader.

I'll use `gist_rainbow` colormap instead of `Set3`. It's quite a garish palette, so I'll set `alpha=0.8`.